index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once 'Autor.php';
+require_once 'ImprimirAutor.php';
+require_once 'Libro.php';
+require_once 'Revista.php';
+require_once 'IPublicable.php';
+
+// Crear Autores
+$autor1 = new Autor("Elena", "White", 1827);
```

Comment:
```suggestion
$autorElenaWhite= new Autor("Elena", "White", 1827);
```
Usar un nombre más descriptivo